js/graph_config.js

```diff
@@ -262,22 +262,34 @@ GraphConfig.load = function(config) {
                     max = Math.max(max, fieldStat.max);
                 }
             }
-
+            
```

Comment:
Stray whitespace. :space_invader:

Reply:
Can you specify what is the whitespace policy?

Trailing whitespace should be erased I guess but what about empty lines which keep the indent between code lines? Most editors insert this kind of indent.

I'm asking because from the file itself I couldn't figure out the exact policy, since there's a mixture of trailing spaces and empty but indented lines.

Reply:
The policy is that no extra whitespace should be added after the last character of any line, including empty lines.

Admittedly, the existing code isn't great when it comes to adherence to coding standards, but that should not be a reason to not be better with new code.

Reply:
This is the coding style that is used for the firmware, so it applies to C, but I think that it makes sense to apply this as well to the non-C projects that are part of Betaflight: https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md

Reply:
Got it. I'll fix. I'll also add `ITERM_RELAX` names.